genrl/environments/suite.py

```diff
@@ -96,3 +109,19 @@ def AtariEnv(
             env = wrapper(env)
 
     return env
+
+
+def MultiagentEnv(env_id: str)-> gym.Env:
+    # load scenario from script
+    scenario = scenarios.load(env_id + ".py").Scenario()
+    benchmark = False
+    # scenario = Scenario()
+    # create world
+    world = scenario.make_world()
+    # create multiagent environment
+    if benchmark:
+        env = MultiAgentEnv(world, scenario.reset_world, scenario.reward, scenario.observation, scenario.benchmark_data, scenario.isFinished)
+    else:
+        env = MultiAgentEnv(world, scenario.reset_world, scenario.reward, scenario.observation, None, scenario.isFinished)
+
+    return GymWrapper(env)
```

Comment:
Do you plan to use vectorised envs later? (That would become multiagent multienv :P)